### PR TITLE
The Ember Times No. 132 - January 24th 2020

### PR DESCRIPTION
**Pull request by amyrlam**
📝 Ideas, feel free to add to list or claim!

- [ ] ~~Setup your Ember Proxy Server for Socket.io https://medium.com/@mudflye/setup-your-ember-proxy-server-for-socket-io-9d88302ceb82~~
- [ ] New RFC: Deprecation Shaking - https://github.com/emberjs/rfcs/pull/565
- [ ] New RFC: URL Manager https://github.com/emberjs/rfcs/pull/570
- [ ] New RFC: Promise-based Initializers https://github.com/emberjs/rfcs/pull/572
- [ ] New RFC: https://github.com/emberjs/rfcs/pull/573
- [ ] New RFC: https://github.com/emberjs/rfcs/pull/574
- [ ] New RFC: https://github.com/emberjs/rfcs/pull/575
- [ ] New RFC: https://github.com/emberjs/rfcs/pull/576
- [ ] New RFC: https://github.com/emberjs/rfcs/pull/578
- [ ] New RFC: https://github.com/emberjs/rfcs/pull/579
- [ ] New RFC: https://github.com/emberjs/rfcs/pull/580
- [x] Ask readers to create Meetup events at EmberConf https://emberconf.com/#/schedule/meetups @MinThaMie 🔒 
  - Maybe suggest ideas based on last year's events? https://github.com/tildeio/emberconf-meetups/blob/master/meetups.yml
- [x] Tracerbench https://twitter.com/Runspired/status/1218669365182074885 (:lock: @amyrlam)
- [x] Related Files Hopper https://twitter.com/suchita009/status/1218310260592496641 (🔒 @chrisrng )
  - On Discord - For all the folks using VSCode as their editor, I have developed the Related Files Hopper extension that will allow you to navigate between related files in your project with just one command. 
Would love to have you guys try it out and hear feedback if any
- [ ] Ember-SortableJS v2 beta release https://twitter.com/luis_vegerano/status/1217940891626889221
  - Isaac - From what I could tell, I think the beta update primarily concerned Octanify-ing the addon. Maybe we can use this as a showcase to encourage addon developers to update their code? https://github.com/SortableJS/ember-sortablejs/pull/13/files
- [x] EmberMap video on RouteInfo's metadata feature https://embermap.com/topics/what-s-new-in-ember/routeinfo-metadata-3-10 (🔒 @ijlee2 )
- [x] EmberConf advertisement with @mansona (🔏 @jessica-jordan)

### Outstanding Team Quotes about Octane from https://github.com/ember-learn/ember-blog/pull/459

- [ ] @MelSumner
- [ ] @kategengler
- [ ] @mixonic
- [ ] @pzuraq

📆 Future stories

- [ ] Write up on current website update status. Call for contributors, explaining the styleguide and which projects contributors can help with. Ping @mansona when this topic is due for release in the Times

🤔 Questions?

See #support-ember-times on Discord or CONTRIBUTING.

✅ Todos:

- Check https://github.com/ember-learn/whats-new-in-emberland (roundup of all the PRs in emberjs and ember-learn repos) for ideas
- Use https://github.com/ember-learn/emberjs-times-tools README
- Check Netlify preview, to ensure display on emberjs.com/blog is good
  - [x] Check buttons
- [x] Crosspost on dev.to, delete Readers Question picture, add Canonical URL 
- [x] Set up tweet for Monday morning on tweetdeck

### CODE REVIEW

File: source/2020-01-24-the-ember-times-issue-132.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+Besides learning about the theory and practice of JAM stack, what makes EmberConf your go-to conference in 2020? Chris shares his perspective on what makes the event so worth being part of:
+
+> The thing I most like about EmberConf is meeting people, chatting in the hallway and the wild plans for the future of Ember that always seem to happen late night over dinner or between two talks.
+
+If you want to learn more about Ember Octane, Empress and JAM, join more than 800 other Ember developers at [EmberConf in Portland, OR, from March 16 - 18, 2020](https://emberconf.com/). Prices for regular attendee tickets start from $449. Get your tickets, before they are sold out, [on the EmberConf website](https://emberconf.com/#/register)!
+<!--alex enable king-queen-->
```
Comment (amyrlam) [inline source/2020-01-24-the-ember-times-issue-132.md]: @jessica-jordan i haven't had much luck with alex disable / enable or ignoring for the whole file, 

it's something i want to look into more, just haven't had time

alex-enable will literally appear
image

just knowledge sharing w/this comment

File: source/2020-01-24-the-ember-times-issue-132.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+What is Tracerbench? Think "Lighthouse CI" but with **statistical rigor** and **more meaningful data**. With [ember-performance-monitoring/tracerbench-compare-action](https://github.com/ember-performance-monitoring/tracerbench-compare-action), you can monitor your web app's performance in CI. Although the library is general enough for any web app, Tracerbench comes finely tuned for benchmarking Ember apps and addons via GitHub Actions.  
+
+Tracerbench's motivation to provide clear, actionable and usable insights into performance deltas is throughly explained in [Tracerbench/tracerbench](https://github.com/TracerBench/tracerbench). There's currently a gap in performance analysis tooling for Ember apps. Developers today struggle to quickly find and analyze performance regressions which would empower them to make quick, iterative changes **within their local development environment**. The current approach for performance analysis is for developers to run a single trace using Chrome Developer Tools. Unfortunately, a single trace varies too much to detect regressions to a web app...unless the regression is quite large.
```
Comment (ijlee2) [inline source/2020-01-24-the-ember-times-issue-132.md]: Typo: throughly should be thoroughly
```suggestion
Tracerbench's motivation to provide clear, actionable and usable insights into performance deltas is thoroughly explained in [Tracerbench/tracerbench](https://github.com/TracerBench/tracerbench). There's currently a gap in performance analysis tooling for Ember apps. Developers today struggle to quickly find and analyze performance regressions which would empower them to make quick, iterative changes **within their local development environment**. The current approach for performance analysis is for developers to run a single trace using Chrome Developer Tools. Unfortunately, a single trace varies too much to detect regressions to a web app...unless the regression is quite large. 
```

Comment (amyrlam) [reply source/2020-01-24-the-ember-times-issue-132.md]: thx!

**Comment by amyrlam** [comment]
merging in

**Pull request merged**